STYLEGUIDE.md

```diff
@@ -43,6 +44,8 @@ Assume the reader is a busy non-native English speaker.
       - Sourcegraph['s] Firefox add-on
       - Sourcegraph['s] Safari extension
 
+- Code intel reserved for describing benefit of using Sourcegraph holistically or from the organization's point of view. If referring to "go to definition" for example, use "code navigation" (not "code intel").
```

Comment:
Whoops, I meant to move the style guide from this repo to https://sourcegraph.com/github.com/sourcegraph/sourcegraph@master/-/blob/doc/dev/style_guide.md, but I didn't push apparently

Reply:
will finish that soon